### ISSUE TITLE
Switch to using new Insert Media dialog

### ISSUE THREAD

**Issue by wpsmort** [semperfiwebdesign/all-in-one-seo-pack#684]
See #683 

Switch to the new Insert Media dialog (or our own) as shown below:

![screen-capture-1](https://cloud.githubusercontent.com/assets/17548525/21031710/ecf2c39e-bd72-11e6-9d0b-e905255bde90.png)

**Comment by michaeltorbert** [comment]
I've reverted this commit back to before. We should put this in a feature branch in case it can't be released for this version.

**Comment by wpsmort** [comment]
@michaeltorbert I found one bug with this.  When you set the OG Image for the homepage under Home Page Settings that should be used on the homepage as the og:image, twitter:image and meta image.  It is correctly being used for the og:image and meta image but not the twitter:image.  The twitter:image is coming from a post where I set the Twitter Image.

To reproduce this, set the Home Image under Home Page Settings and verify its being used correctly on the homepage for all three. Then create a post and add a Twitter image in the Social Settings tab. Now check the homepage and the twitter:image should now be incorrect.
